config.py

```diff
@@ -228,7 +274,6 @@ def build_config(_config_file):
                         'TARGET_SOCK': (gethostbyname(config.get(section, 'TARGET_IP')), config.getint(section, 'TARGET_PORT')),
                         'TARGET_IP': gethostbyname(config.get(section, 'TARGET_IP')),
                         'TARGET_PORT': config.getint(section, 'TARGET_PORT'),
-                        'BOTH_SLOTS': config.getboolean(section, 'BOTH_SLOTS'),
```

Comment:
Same again, I'd like to have this line in the config.py preserved, as it supports unit-to-unit (i.e. private) calls, especially over OpenBridge, as I currently understand it.